.github/workflows/ci.yml

```
@@ -2,167 +2,132 @@ name: CI
     branches: [ main ]
 
 jobs:
-  test:
+  basic-checks:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ['3.10', '3.11', '3.12', '3.13']
-
     steps:
     - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)